src/lib.rs

```diff
@@ -56,8 +57,6 @@ use std::fmt;
 use std::error;
 use libc::PATH_MAX;
 
-pub type Result<T> = result::Result<T, Error>;
```

Comment:
My confusion stemmed from moving this into the `errno` module. What's the rationale?